### PR TITLE
Upgrade CXF to 3.4.5

### PR DESCRIPTION
**Pull request by wborn**
It would be nice when CXF 3.4.5 is used as dependency, so we don't have to add some workarounds for this in openHAB (https://github.com/openhab/openhab-core/pull/2532). :slightly_smiling_face:

### CODE REVIEW

File: integrations/cxf-jettison/cxf-jettison-itest/itest.bndrun [hunk 1]
```diff
@@ -52,22 +52,27 @@ java.9.plus.reqs: \
 
 -runbundles: \
 	com.fasterxml.woodstox.woodstox-core;version='[6.2.6,6.2.7)',\
+	com.sun.xml.bind.jaxb-osgi;version='[2.3.3,2.3.4)',\
```
Comment (rotty3000) [inline integrations/cxf-jettison/cxf-jettison-itest/itest.bndrun:55]: Looks like you resolved on Java higher than 8. Can you re-resolve while running the build with Java 8 JDK?

**Review by rotty3000 — CHANGES_REQUESTED** [review]
Looks like you need to revert all resolution results and redo it using Java 8.

maven is very good about respecting `JAVA_HOME` env var so that is the easiest if you don't want to change your default.

```bash
JAVA_HOME=/usr/lib/jvm/distro-8 mvn clean verify
```

File: integrations/cxf-jettison/cxf-jettison-itest/itest.bndrun (hunk 1 above)
Comment (rotty3000) [reply integrations/cxf-jettison/cxf-jettison-itest/itest.bndrun:55]: oh, and please rebase on HEAD.

Comment (wborn) [reply integrations/cxf-jettison/cxf-jettison-itest/itest.bndrun]: Yes I used Java 11 before. But now I've resolved the bundles using Java 8.

**Review by rotty3000 — APPROVED** [review]

**Pull request merged**